packages/wdio-browserstack-service/src/types.ts

```diff
@@ -57,6 +57,18 @@ export interface BrowserstackConfig {
      * For e.g. buildName, projectName, BrowserStack access credentials, etc.
      */
     testObservabilityOptions?: TestObservabilityOptions;
+    /**
+    * Set this to true to enable BrowserStack Accessibility Automation which will
+    * automically conduct accessibility testing on your pre-existing test builds
+    * and generate health reports which can be viewed in the Accessibility dashboard.
+    * @default false
+    */
+    accessibility?: boolean | string;
```

Comment:
The description suggests it is a boolean only
```suggestion
    accessibility?: boolean;
```

Reply:
@christian-bromann We are expecting both boolean and string value from users. Have handled both the cases in several parts/functions in the code.

Reply:
addressed this comment also to make it consistent with other product flag.